.travis.yml

```diff
@@ -17,11 +17,14 @@ env:
   - NJOBS=2
   # system is == 4.02.3
   - COMPILER="system"
+  # - COQ_GIT=git://scm.gforge.inria.fr/coq/coq.git
+  - COQ_GIT=https://github.com/ejgallego/coq.git
```

Comment:
No, this patch will be dropped once we agree on a merge. We have a better overlay system on the works (one that actually picks the proper overlay based on `TRAVIS_PR` etc... but it is not ready yet so we have to resort to workaround.